app/lib/utils/queryMapper.js

```diff
@@ -41,6 +44,12 @@ function mapServiceChoice(query) {
   if ((query.symptoms) && (query.symptoms === constants.SYMPTOMS.yes)) {
     return constants.SERVICE_CHOICES.symptoms;
   }
+  if ((query.age) && (query.age === constants.AGE['16to25'])) {
```

Comment:
Could this be simplified so all the age checks are together within one `if (query.age) {` ?